app/views/admin/claims/show.html.erb

```diff
@@ -1,5 +1,11 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <% if @claim.payroll_gender_missing? %>
+      <div class="govuk-body-l govuk-flash__notice">
+        This claim cannot be approved, the payroll gender is missing and it will need to be referred
```

Comment:
This claim cannot be approved, the payroll gender is missing and ~it~ the claim will need to be referred